int8 robot_num
int8 run	#0 wait 1 running
int8 path_num	#0 none